docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -39,342 +37,123 @@ Devices with Intel® GNA support:
 
 > **NOTE**: On platforms where Intel® GNA is not enabled in the BIOS, the driver cannot be installed, so the GNA plugin uses the software emulation mode only.
 
-## Intel® GNA Generational Differences
+### Drivers and Dependencies
 
```

Comment:
On [getting started](https://docs.openvino.ai/nightly/get_started.html) page we have `ADDITIONAL CONFIGURATIONS`. Looks like it perfectly matches to this paragraph and we need to move this out to an additional page `Configurations for GNA`